components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { HTMLAttributes } from 'react';
+import { cnb } from 'cnbuilder';
+
+type TimelineProps = HTMLAttributes<HTMLDivElement>;
+
+export const Timeline = ({
+  children,
+  className,
+  ...props
+}: TimelineProps) => {
+
+  return (
+    <div
+      {...props}
+      className={cnb('relative w-full grid gap-y-450', className)}
+    >
+      {children}
+      <div className="hidden md:block w-2 absolute top-0 left-1/2 -ml-1 h-full origin-top bg-purple" />
+    </div>
+  );
+};
```

Comment:
I took the Timeline component out of the Grid since it doesn't really have much to do with the Grid. It's now the wrapper for the timeline and it takes the TimelineItem components as children 😄